salad/operator/scripts/distribute_funds.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import sys
+from enigma_docker_common.faucet_api import request_coins
+
+
+def main():
+    addresses = sys.argv[1:]
+    for address in addresses:
+        print(f'distributing ether to {address}')
+        request_coins('http://contract.reuven.services.enigma.co:8001', address, 'ether')
```

Comment:
replace reuven with os.environ['enigma_namespace']

Reply:
I ended up implementing this by passing it in as an environment variable, which is replaced dynamically in the kubernetes environment.

Implemented in:
https://github.com/enigmampc/kubernetes/pull/26
5755845 
https://github.com/enigmampc/salad/pull/53 (commit `f4e726f`)

Reply:
Ended up closing PR 26 in kubernetes and replacing commit 5755845 with 4f28dd5